spec/rubocop/cop/lint/empty_conditional_body_spec.rb

```diff
@@ -24,19 +24,60 @@
   it 'registers an offense for missing `elsif` body' do
     expect_offense(<<~RUBY)
       if condition
-        do_something
-      elsif other_condition
-      ^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
+        do_something1
+      elsif other_condition1
+      ^^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
+      elsif other_condition2
+        do_something2
       end
     RUBY
 
     expect_correction(<<~RUBY)
       if condition
+        do_something1
+      elsif other_condition2
+        do_something2
+      end
+    RUBY
+  end
+
+  it 'registers an offense for missing `if` and `elsif` body' do
+    expect_offense(<<~RUBY)
+      if condition
+      ^^^^^^^^^^^^ Avoid `if` branches without a body.
+      elsif other_condition1
+      ^^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
+      elsif other_condition2
+        do_something
+      end
+    RUBY
+
+    expect_correction(<<~RUBY)
+      if other_condition2
         do_something
       end
     RUBY
   end
 
+  it 'registers an offense for missing all branches of `if` body' do
+    expect_offense(<<~RUBY)
+      if condition
+      ^^^^^^^^^^^^ Avoid `if` branches without a body.
+      elsif other_condition
+      ^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
+      end
+      if condition
+      ^^^^^^^^^^^^ Avoid `if` branches without a body.
+      elsif other_condition1
+      ^^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
+      elsif other_condition2
+      ^^^^^^^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
+      end
```

Comment:
Wouldn't it be better to split L64-L68 and L69-L75 into different test cases?

Reply:
That aside, I watch the repository so review request is unneeded after PR update.